.github/workflows/deploy.yml

```diff
@@ -162,7 +162,7 @@ jobs:
           mv ./other/.dockerignore ./.dockerignore
 
       - name: 🎈 Setup Fly
-        uses: superfly/flyctl-actions/setup-flyctl@v1.4
+        uses: superfly/flyctl-actions/setup-flyctl@master
```

Comment:
for this one, I tried different syntax variations with the pinned version `v1.5` but for some reason it said that it doesn't exist, even though the repo for the action says it does exist. Maybe someone can get it working?

Reply:
I don't feel comfortable using `master`. There is a tag for [`v1.5`](https://github.com/superfly/flyctl-actions/releases/tag/1.5). Can you share a reproduction of this not working?

Reply:
![image](https://github.com/epicweb-dev/epic-stack/assets/30846026/f9f94c9e-5a39-4529-9134-da2f07e33a78)

Reply:
Ah, good news, I tried removing the `v` from it, and it works. Dang, I thought I had already tried that, woopsie. Maybe give it a spin yourself before merging in to make sure though?